pymagicc/api.py

```diff
@@ -230,6 +230,35 @@ def set_config(
 
         return data
 
+    def update_config(
+        self, filename="MAGTUNE_PYMAGICC.CFG", top_level_key="nml_allcfgs", **kwargs
+    ):
+        """
+        Updates a configuration file for MAGICC
+
+        Updates the contents of a fortran namelist in the run directory, creating a new namelist if none exists.
+
+        # Parameters
```

Comment:
Need to move to numpy style sphinx docstrings https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard